tools/title2sentance.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+
+import sys
+import argparse
+
+keep_words = [
+    'Web Console',
+    'Red Hat',
+    'Docker',
+    'CPU',
+    'IP',
+    'VNC',
+    'VM',
+    'LVM',
+    'STP',
+    'SSH',
```

Comment:
WE maybe could have a logic that if single word is all caps, just keep it all caps...